tagging/settings.py

```diff
@@ -3,11 +3,10 @@
 which enforces default settings when the main settings module does not
 contain the appropriate settings.
 """
-from django.conf import settings
```

Comment:
I was having problems importing this file with this line it. So, I stopped. I don't really like the solution, but I couldn't find a way around it.